sopel/coretasks.py

```diff
@@ -131,14 +131,16 @@ def startup(bot, trigger):
     bot.memory['retry_join'] = dict()
 
     channels = bot.config.core.channels
+    if not channels:
+        LOGGER.info('No initial channels to JOIN.')
```

Comment:
Shouldn't this `return`? If there are no channels, it's silly to log this _and_ "Joining 0 channels" later.

Reply:
Oh. I forgot to use a `elif` after that... >.<